chia-protocol/src/bytes.rs

```diff
@@ -48,6 +54,33 @@ impl Streamable for Bytes {
     }
 }
 
+#[cfg(feature = "py-bindings")]
+impl ToJsonDict for Bytes {
```

Comment:
these were moved verbatim. They used to be in the same translation unit as the trait definition